src/main/resources/application.yml

```diff
@@ -32,6 +32,8 @@ learnerRecord:
 csrs:
   serviceUrl: ${REGISTRY_SERVICE_URL:http://localhost:9002}
   deleteUrl: "${csrs.serviceUrl}/civilServants/%s/delete"
+  organisationListUrl: "${csrs.serviceUrl}/organisationalUnits/tree"
+  AddOrganisationReportingPermissionUrl: "${csrs.serviceUrl}/organisationalUnits/addOrganisationReportingPermission/%s"
 
 notifications:
```

Comment:
The current formatting is non-capitalized first letter as in addOrganisationReportingPermissionUrl